src/components/Trend/Trend.js

```diff
@@ -63,13 +63,13 @@ class Trend extends Component {
     const { autoDraw, autoDrawDuration, autoDrawEasing } = this.props;
 
     const path = document.querySelector('.trend-line path');
-    path.classList.add('animate');
+    path.setAttribute('class', 'trend-line animate');
```

Comment:
I believe we have polyfills for doms that works well with using classList. If that's not available we can easily copy the polyfills from mozillas documentation with the source here and add it to the source's library so it's easily accessible.

Class lists are known to be faster that there get and setAttributes aliases